### DISCUSSION TITLE
Marp Usecase - Scientific Presentations

### DISCUSSION THREAD

**Discussion by cheind** [marp-team/marp#231] [Show and tell]
Hey,

I just wanted to say that Marpit is great for scientific presentations! I only recently came across it, but decided to use it for my next presentation on a machine learning topic. I've made everything open source - including the presentation. So if you need a showcase, here's one:

https://github.com/cheind/autoregressive/blob/main/etc/presentation/autoregressive_presentation.pdf
https://github.com/cheind/autoregressive

**Comment by cunhapaulo** [comment]
You´ve created a very beautiful and useful theme!
Congrats!
